controller/Tools.php

```diff
@@ -112,20 +141,23 @@ public function pauseActiveExecutions()
 
         $this->returnJson([
             'success' => !count($notPaused),
-            'message' => count($paused) . ' ' . __('sessions paused'),
-            'processed' => $paused,
-            'unprocessed' => $notPaused
+            'data' => [
+                'message' => count($paused) . ' ' . __('sessions paused'),
+                'processed' => $paused,
+                'unprocessed' => $notPaused
+            ]
         ]);
     }
 
     /**
      * @return \DateInterval
      */
-    private function getInterval()
+    private function getInterval($param)
```

Comment:
For this one, I do feel it's best to retrieve the request parameter from the endpoint. It has better modularity. For instance, getInterval is now able to be unit tested without having to make an http request.